romancal/regtest/test_wfi_saturation.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pytest
+
+from romancal.stpipe import RomanStep
+from romancal.step import SaturationStep
+import roman_datamodels as rdm
+from .regtestdata import compare_asdf
+
+@pytest.mark.bigdata
+def test_saturation_image_step(rtdata, ignore_asdf_paths):
+    # Testing retrieval of best ref file for image data,
+    # and creation of a ramp file with CRDS selected saturation file applied.
+
+    rtdata.get_data("WFI/image/ramp.asdf")
+    rtdata.input = "ramp.asdf"
+
+    # Test CRDS
+    step = SaturationStep()
+    model = rdm.open(rtdata.input)
+    ref_file_path = step.get_reference_file(model, "saturation")
+    ref_file_name = os.path.split(ref_file_path)[-1]
+
+    assert "roman_wfi_saturation" in ref_file_name
```

Comment:
I don't quite understand this : doesn't this merely verify that *a* "roman_wfi_saturation" file (perhaps the only one currently in CRDS ? ) was retrieved ? If so an additional comment would help.

Reply:
Added comments.